Build error - Can't resolve './components/Kanban'

Hello,

With Webpack 3.7, I have the following build error:

```
ERROR in ./node_modules/vue-kanban/src/plugin.js
Module not found: Error: Can't resolve './components/Kanban'
```

I managed to fix the issue by changing  the first ling of `src/plugin.js` from

`import Kanban from './components/Kanban';`

to

`import Kanban from './components/Kanban.vue';`

Could you please push the change and tag a new version? If you want I can create a pull request with the changes.

Thank you :)

--- a/src/plugin.js
+++ b/src/plugin.js
@@ -1,4 +1,4 @@
-import Kanban from './components/Kanban';
+import Kanban from './components/Kanban.vue';
 
 export default {
   install(vue) {

